packages/editor/src/client/atoms/dateTimePicker.tsx

```diff
@@ -19,6 +28,7 @@ export interface DateTimePickerProps {
 
   dateRanges?: DateTimePreset[]
   timeRanges?: DateTimePreset[]
+  helpInfo?: string
```

Comment:
Can this be added external to the datePicker component? I'm not sure if it's necessary to be part of this, or would it make more sense to have the info label as part of the publish panel? Alternatively maybe the label can be set as children to allow to pass the label text with any custom icons inside. Not sure if this is possible.